docs/INSTALLATION.md

```diff
@@ -2,20 +2,11 @@
 
 ## Installing Dart
 
-### Automatic installation instructions (recommended)
-* Follow the [instructions for installation on the Dart website](https://www.dart.dev/tools/sdk).
+Instructions to install the Dart SDK can be found on the [Get Dart][get-dart] page.
```

Comment:
I would suggest adding, "can be found on the official Date website's Get Dart page."

Just so folks don't think it's our page